spec/api-app-spec.js

```diff
@@ -796,17 +796,21 @@ describe('app module', () => {
     })
   })
 
-  // TODO(marshallofsound): [Ch66] Failed on Windows x64 + ia32 on CI, passes locally
-  xdescribe('getAppMetrics() API', () => {
+  describe('getAppMetrics() API', () => {
     it('returns memory and cpu stats of all running electron processes', () => {
       const appMetrics = app.getAppMetrics()
       expect(appMetrics).to.be.an('array').and.have.lengthOf.at.least(1, 'App memory info object is not > 0')
 
       const types = []
       for (const {memory, pid, type, cpu} of appMetrics) {
         expect(memory.workingSetSize).to.be.above(0, 'working set size is not > 0')
-        expect(memory.privateBytes).to.be.above(0, 'private bytes is not > 0')
-        expect(memory.sharedBytes).to.be.above(0, 'shared bytes is not > 0')
+
+        // windows causes failures here due to CI server configuration
```

Comment:
This is an improvement over an unconditionally disabled  test -- but Is there no way to fix the CI server configuration instead s.t. all of these expect() calls can be enabled?

Reply:
After discussion with @codebytere, putting the win32 test question onto the backlog at https://github.com/electron/electron/issues/14003